scripts/uberenv/packages/serac/package.py

```diff
@@ -96,7 +96,7 @@ class Serac(CMakePackage):
     depends_on('py-sphinx', when="+devtools")
 
     # Libraries that support +debug
-    debug_deps = ["mfem@4.1.0~shared+hypre+metis+superlu-dist+lapack+mpi",
+    debug_deps = ["mfem@4.1.0p1~shared+hypre+metis+superlu-dist+lapack+mpi",
```

Comment:
Things like this are the stuff of nightmares. Thank you for figuring it out!

Reply:
Credit is on @white238. He knew the right person to ask :)